render/helpers_test.go

```diff
@@ -238,6 +244,14 @@ func TestDatePeriodFormat(t *testing.T) {
 
 }
 
+func TestDateFormatYYYYMMDDNoSlash(t *testing.T) {
+	Convey("Date format returns human readable string", t, func() {
```

Comment:
I maybe wouldn't opt to use the word 'readable' here 😅

Reply:
👍